osu.Game.Rulesets.Osu/Difficulty/OsuDifficultyCalculator.cs

```diff
@@ -60,6 +61,7 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
                 Mods = mods,
                 AimStrain = aimRating,
                 SpeedStrain = speedRating,
+                FlashlightRating = flashlightRating,
```

Comment:
As we discussed in the pp discord, let's make FL affect overall star rating as well

Reply:
Done in a separate PR -> https://github.com/ppy/osu/pull/14753